lib/const.rb

```diff
@@ -34,6 +34,14 @@
   HOME = CREW_PREFIX + ENV['HOME']
 end
 
+if ENV['CREW_CACHE_DIR'].to_s == ''
+  CREW_CACHE_DIR = HOME + '.cache/crewcache'
```

Comment:
Do we need `'/.cache/crewcache'`?

Reply:
~/.cache seems like it could be used by multiple programs, so let's not clobber?

Reply:
Do we need a ```/``` before ```./cache```?

Reply:
Yeah, that was my question.

Reply:
Thanks for getting me there. :)  Fixed.